packages/create-bison-app/template/components/auth/LoginForm.tsx

```diff
@@ -25,8 +25,8 @@ export function LoginForm() {
   const {
     register,
     handleSubmit,
-    formState: { errors, isValid, isSubmitted },
-  } = useForm<LoginFormData>({ mode: 'onChange' });
+    formState: { errors, isValid },
+  } = useForm<LoginFormData>();
```

Comment:
We don't want to validate until the form is submitted (it's annoying to see an error message while you're typing).